rustls/src/msgs/message.rs

```diff
@@ -459,6 +459,149 @@ impl<'a> BorrowedPlainMessage<'a> {
     }
 }
 
+#[derive(Debug, Clone)]
+/// A collection of borrowed plaintext slices
+pub enum BorrowedPlainPayload<'a> {
+    Empty,
+    /// A single byte slice. This allows only one redirection instead of several.
+    Single(&'a [u8]),
+    /// A collection of chunks (byte slices)
+    /// and inclusive-exclusive cursors to single out a slice of bytes.
+    /// [BorrowedPayload] assumes three invariants:
```

Comment:
I think this should be `BorrowedPlainPayload` after the rename (_Sorry, I think I'm the one that missed that when doing the rename!_)